### PR TITLE
change network config format, and update layer act

### PR DESCRIPTION
**Pull request by warshallrho**
### Checklist
- [x] I've tested that my changes are compatible with the latest version of Tensorflow.
- [x] I've read the [Contribution Guidelines](https://github.com/tensorlayer/tensorlayer/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Description

Please review.

### CODE REVIEW

File: tensorlayer/layers/core.py [hunk 1]
```diff
@@ -17,6 +17,15 @@
 
 _global_layer_name_dict = {}  # TODO: better implementation?
 
+_act_dict = {
+    "relu": tf.nn.relu,
+    "relu6": tf.nn.relu6,
+    "leaky_relu": tf.nn.leaky_relu,
```
Comment (zsdonghao) [inline tensorlayer/layers/core.py]: when act = 'lrelu0.2'  or 'leaky_relu0.2'  act = lambda x : tf.nn.leaky_relu(x, 0.2'

File: tensorlayer/layers/core.py [hunk 2]
```diff
@@ -84,6 +94,13 @@ def __init__(self, name=None, *args, **kwargs):
                 _global_layer_name_dict[name] = 0
 
         self.name = name
+        if haveact:
```
Comment (zsdonghao) [inline tensorlayer/layers/core.py]: if isfun(act):
    self.act = act
elif act == None:
    self.act = lambda x: x
elif instance(act, str):
    self.act = str2act(act)
else:
   raise Exception("unsupported act {}".format(act))

File: tensorlayer/layers/core.py [hunk 3]
```diff
@@ -17,6 +17,22 @@
 
 _global_layer_name_dict = {}  # TODO: better implementation?
 
+_act_dict = {
+    "relu": tf.nn.relu,
+    "relu6": tf.nn.relu6,
+    "leaky_relu": tf.nn.leaky_relu,
+    "lrelu0.2": tf.nn.leaky_relu,
```
Comment (zsdonghao) [inline tensorlayer/layers/core.py]: this is incorrect ..  need to parse the number from `lrelu222.222: lambda x : tf.nn.leaky_relu(x, 222.222)

Comment (warshallrho) [reply tensorlayer/layers/core.py]: got it

File: tensorlayer/layers/core.py (hunk 1 above)
Comment (warshallrho) [reply tensorlayer/layers/core.py]: @zsdonghao fixed

File: tensorlayer/layers/core.py (hunk 3 above)
Comment (warshallrho) [reply tensorlayer/layers/core.py]: @zsdonghao fixed

**Pull request merged**